polaris-react/src/components/TextField/TextField.tsx

```diff
@@ -201,6 +204,7 @@ export function TextField({
   id: idProp,
   role,
   step,
+  stepperShownOnInteraction,
```

Comment:
Can we set this to false?
```suggestion
  stepperShownOnInteraction = false,
```